packages/drivers/local-driver/src/localDocumentDeltaConnection.ts

```diff
@@ -79,9 +79,13 @@ export class LocalDocumentDeltaConnection extends DocumentDeltaConnection {
 
 	/**
 	 * Submits a new signal to the server
+	 *
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: any): void {
-		this.emitMessages("submitSignal", [[message]]);
+	public submitSignal(content: any, targetClientId?: string): void {
+		// TODO: Add targetClientId support
```

Comment:
What is the work to support targetClientId? (We should always try to reference a work item number in TODOs.)

Reply:
Will look into this in a separate future PR. Removed this change for now.